pkg/daemon/update.go

```diff
@@ -2292,13 +2295,16 @@ func (dn *CoreOSDaemon) applyLegacyOSChanges(mcDiff machineConfigDiff, oldConfig
 		if dn.nodeWriter != nil {
 			dn.nodeWriter.Eventf(corev1.EventTypeNormal, "OSUpgradeApplied", "OS upgrade applied; new MachineConfig (%s) has new OS image (%s)", newConfig.Name, newConfig.Spec.OSImageURL)
 		}
-	} else { //nolint:gocritic // The nil check for dn.nodeWriter has nothing to do with an OS update being unavailable.
+	} else { //nolint:gocritic // The nil check for dn.nodeWriter has nothing to do with an OS update begit sting unavailable.
```

Comment:
I assume this is an auto correction? :)

Reply:
...I think was doing a git stash in the wrong window!